aldryn_config.py

```diff
@@ -93,6 +105,9 @@ def to_settings(self, data, settings):
 
         settings['CMS_PERMISSION'] = data['permissions_enabled']
 
+        settings['CMS_CACHE_DURATIONS']['content'] = data['cms_content_cache_duration']
+        settings['CMS_CACHE_DURATIONS']['menus'] = data['cms_menus_cache_duration']
```

Comment:
since the field is ``required=False`` you need to handle the case where it is not set (don't set the setting at all. Or hardcode the default as fallback.).